python/storage-cleaner/src/appwrite_service.py

```
@@ -0,0 +1,73 @@
+        ]
+
+        deleted_files_count = 0
+        failed_files = []
+
+        while True:
+            try:
+                response = self.storage.list_files(bucket_id, queries)
+            except Exception as e:
+                raise RuntimeError(
+                    f"Failed to list files from bucket {bucket_id}: {str(e)}"
+                ) from e
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)